remote_challenge_evaluation/remote_evaluation_util.py

```diff
@@ -0,0 +1,89 @@
+import os
+import requests
+from eval_ai_interface import EvalAI_Interface
+
+
+class RemoteEvaluationUtil:
+    def __init__(self):
+        # TODO: Populate the variables for the evaluation
+        self.auth_token = ""  # Go to EvalAI UI to fetch your auth token
+        self.evalai_api_server = ""  # For staging server, use -- https://staging.eval.ai; For production server, use -- https://eval.ai
+        self.queue_name = ""  # Check Manage Tab of challenge for queue name
+        self.challenge_pk = ""  # Check Manage Tab of challenge for challenge PK
```

Comment:
Can we fetch these variable values from environment variables using `os.environ["XYZ"]`. You can mention in docs that hosts need to set these environment variables